example/src/main/java/io/github/btkelly/gandalf/example/utils/MockWebServerUtil.java

```diff
@@ -65,7 +67,7 @@ public void run() {
                     MockResponse mockResponse = new MockResponse();
                     mockResponse.setResponseCode(200);
                     mockResponse.setBody(mockBootstrapJsonBody);
-                    mockResponse.setBodyDelay(4, TimeUnit.SECONDS);
+                    mockResponse.setBodyDelay(2, TimeUnit.SECONDS);
```

Comment:
Nothing really, I just wanted to show that what could be done like a progress spinner while the call is loading. 4 seemed too long while I was testing, got sick of waiting.